### ISSUE TITLE
Download as ARC is not an ARC

### ISSUE THREAD

**Issue by AngelaKranz** [usadellab/plant-adaptation-hub#41]
If you say that it is downloaded as an ARC, it should actually comply with the relevant ARC specifications.

**Comment by ataulhaleem** [comment]
yes its not arc but RO-crate i have added some documentation.. more coming

**Comment by AngelaKranz** [comment]
This has still not changed. You can not call it ARC if it is not fulfilling the respective specifications.
